Wireframe/index.html

```diff
@@ -1,13 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <link rel="stylesheet" href="/Wireframe/css/style.css">
+</head>
+
+<body>
+    <header>
+        <section class="title">
```

Comment:
What is this section doing that the plain h1 is not? Can you talk through your decision?

Reply:
I agree with you. Using h1 would have had the same effect since it, like Section, is also a block-level element.